bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,40 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        output_format='regex',
+        output_regex=r'(^(.*/)*)([^/]*): .*UNKNOWN$',
```

Comment:
(we maybe need some regex composition library, where you could do something like `regexes.path + r': .*UNKNOWN$'`